utils/travis-build-repo-internal.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -l
```

Comment:
No, but I think this ensures that it gets run in a login shell even if we invoke it as `./$REPO_PATH/travis-build-repo-internal.sh` instead of `bash -l $REPO_PATH/travis-build-repo-internal.sh`, so I think it's best to keep both.